src/features/usage-consolidation-accordion/usage-consolidation-accordion.js

```diff
@@ -0,0 +1,102 @@
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { FormattedMessage } from 'react-intl';
+
+import { useStripes } from '@folio/stripes/core';
+import {
+  Accordion,
+  Headline,
+} from '@folio/stripes/components';
+
+import Toaster from '../../components/toaster';
+import { getUsageConsolidation as getUsageConsolidationAction } from '../../redux/actions';
+import { selectPropFromData } from '../../redux/selectors';
+
+const propTypes = {
+  getUsageConsolidation: PropTypes.func.isRequired,
+  headerProps: PropTypes.object,
+  id: PropTypes.string.isRequired,
+  isOpen: PropTypes.bool,
+  onToggle: PropTypes.func.isRequired,
+  section: PropTypes.node.isRequired,
+  usageConsolidation: PropTypes.shape({
+    data: PropTypes.object,
+    errors: PropTypes.array,
+    isFailed: PropTypes.bool.isRequired,
+  }).isRequired,
+};
+
+const UsageConsolidationAccordion = ({
+  getUsageConsolidation,
+  headerProps,
+  id,
+  isOpen = true,
+  onToggle,
+  usageConsolidation,
+  section: Section,
+}) => {
+  const stripes = useStripes();
+
+  const canViewUsageConsolidation = stripes.hasPerm('ui-eholdings.costperuse.view');
+
+  useEffect(() => {
+    if (canViewUsageConsolidation) {
+      getUsageConsolidation();
+    }
+  }, [canViewUsageConsolidation, getUsageConsolidation]);
+
+  const getToastErrors = () => {
+    const { errors } = usageConsolidation;
+
+    return errors.map((error) => ({
+      message: error.title,
+      type: 'error',
+    }));
+  };
+
+  const getUsageConsolidationAccordionHeader = () => {
+    return (
+      <Headline
+        size="large"
+        tag="h3"
+      >
+        <FormattedMessage id="ui-eholdings.usageConsolidation" />
+      </Headline>
+    );
+  };
+
+  if (usageConsolidation.isFailed || canViewUsageConsolidation) {
+    return null;
+  }
+
+  return (
+    <>
+      {usageConsolidation.data &&
```

Comment:
`usageConsolidation.data` is an object and in all cases it will be true, so you need to check if `data` has `credentialsId` property
```suggestion
      {usageConsolidation.data?.credentialsId &&
```